js/storage.js

```diff
@@ -0,0 +1,22 @@
+const form1 = document.querySelector('#contact');
+form1.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const obj = {
+    fullName: document.querySelector('#name').value,
+    emailAdress: document.querySelector('#email').value,
+    message: document.querySelector('#message').value,
+  };
+
+  localStorage.setItem('text', JSON.stringify(obj));
+});
+
+const retrievedObject = localStorage.getItem('text');
+const getValue = JSON.parse(retrievedObject);
+
+window.addEventListener('load', () => {
+  if (localStorage.getItem('text')) {
+    document.querySelector('#name').value = getValue.fullname;
```

Comment:
- [ ] A massive welldone for implementing the data persistence to the localStorage and retrieving it as well 👏. 

However, a slight bug 🐛 occurs when you try to retrieve `fullname` on page reload. This is because the key is actually `fullName` and not `fullname` so you get `undefined`. Please make this change in order to get approved 👍 
![image](https://user-images.githubusercontent.com/46839548/210794988-34b87c31-6eec-4a80-8319-990237ffbb11.png)